admin/src/components/Processors/EditProcessor.js

```diff
@@ -79,9 +79,9 @@ export class EditProcessor extends React.Component {
 }
 
 const EditProcessorForm = props => {
-  const { errors, values, ...formProps } = props;
+  const { errors, values, isLoading, ...formProps } = props;
```

Comment:
👍

Reply:
Yeah, that was causing console errors when passing `formProps` down